### PR TITLE
#28 basic CustomerContainer

### PR DESCRIPTION
**Pull request by joshxg**
#28 

- Pretty much a copy paste of the template in the previous PR
- Also just added this to the host
- Wanted to get the boiler plate out of the way

### CODE REVIEW

**Comment by joshxg** [comment]
Ugh. Accidental commits and a bit of a mess reverting them! Sorry

File: packages/customers/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "//": "${UPDATE} - This package name should be updated",
```
Comment (mark-prins) [inline packages/customers/package.json:2]: should it though?

File: packages/customers/src/CustomerContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React, { FC } from 'react';
+
+// This is a remote service which is exposed by this package.
+// This component can be imported by another remote who declares
+// this package as a remote in their webpack config and imported through:
+// React.lazy(() => import("template/service"))
```
Comment (mark-prins) [inline packages/customers/src/CustomerContainer.tsx:6]: ah, from the template? great idea

**Review by mark-prins — APPROVED** [review]
👍  that looks good!

**Pull request merged**